controllers/sriovnetworknodepolicy_controller.go

```diff
@@ -152,6 +157,47 @@ func (r *SriovNetworkNodePolicyReconciler) SetupWithManager(mgr ctrl.Manager) er
 		Complete(r)
 }
 
+func (r *SriovNetworkNodePolicyReconciler) syncOvnNetAttachDefs(pl *sriovnetworkv1.SriovNetworkNodePolicyList) error {
+	logger := r.Log.WithName("syncOvnNetAttachDefs")
+	logger.Info("Start to sync default ovn net-attach-defs")
+
+	for _, p := range pl.Items {
+		if p.Name == "default" {
+			continue
+		}
+		if p.Spec.EswitchMode == sriovnetworkv1.ESWITCHMODE_SWITCHDEV {
+			raw, err := p.RenderNetAttDef()
```

Comment:
we don't have to address it now, but I think there is going to be several cni's that can leverage the switchdev config. 
option 1 - is that we have a flag that skip the  RenderNetAttDef
option 2 - is that we had network policy to each cni
or maybe we need both options. 
What do you think?

Reply:
I agree that we need distinguish switchdev for different CNI cases. I only add switchdev conditional check for ovs hw offload net-attach-def is because it is the only enabled feature atm for switchdev.

Regarding the options, I'd like to see if it's possible to do auto detection of whether a PF is enslaved as OVS port (ovs hw offload) or standalone/enslaved in linux bridge (accelerated bridge). If it's not something detectable, having a flag (API change) would be desired in my opinion.